### DISCUSSION TITLE
How to include / exclude Javascript / css?

### DISCUSSION THREAD

**Discussion by geoidesic** [zotonic/zotonic#3455] [Q&A] [answered]
I've used the zotonic_exampe template. I notice that changes in `_js_include.tpl` don't seem to have any effect, even though this file is included in the `base.tpl`. 

1. Do I need to write a module to strap in SvelteJs?
2. Why are changes in `_js_include.tpl` not being included?
3. I read the colonic docs but it's not clear how to interact with it, also it seems to be being added indirectly by a module.
4. I tried removing bootstrap from `base.tpl` but then the site completely vanishes, doesn't render at all (I don't want bootstrap)
5. I notice a file `_js_include_cotonic.tpl`, however, making it an empty file does not prevent colonic from being included in the headers (because of point 3 above, i.e. module `zotonic_mod_base`)
6. What if I don't want all the zotonic modules included by https://hex.pm/packages/zotonic_apps? (e.g. bootstrap)
7. Are they apps or modules? I'm confused... the naming seems to indicate both.

**Comment by geoidesic** [comment, marked as answer]
I've figured this out. I'll do a write up on it when I can.

**Reply by mworrell** [reply]
Great!
